### PR TITLE
update Acosh spec

### PR DESCRIPTION
**Pull request by pelszkow**
### Details:
 - *update Acosh spec*

### Tickets:
 - *45878*

### CODE REVIEW

**Review by jdanieck — APPROVED** [review]

File: docs/ops/arithmetic/Acosh_3.md [hunk 1]
```diff
@@ -6,32 +6,28 @@
...
+\f[
+a_{i} = acosh(a_{i})
+\f]
+
+**Attributes**: *Acosh* operation has no attributes.
 
 **Inputs**
 
-* **1**: A tensor of type *T*. **Required.**
+* **1**: A tensor of type *T* and arbitrary shape. **Required.**
```
Comment (sdurawa) [inline docs/ops/arithmetic/Acosh_3.md:19]: Is input tensor provided in radians or degrees?

Comment (pelszkow) [reply docs/ops/arithmetic/Acosh_3.md:19]: Which trigonometric operator provide such information?

Comment (sdurawa) [reply docs/ops/arithmetic/Acosh_3.md:19]: I saw it in tan op spec review (https://github.com/openvinotoolkit/openvino/pull/6567/files)

Comment (pelszkow) [reply docs/ops/arithmetic/Acosh_3.md:19]: Won't do if this is only in one trigonometric op. It should be one PR which update this in all.

**Review by sdurawa — APPROVED** [review]

File: docs/ops/arithmetic/Acosh_3.md [hunk 2]
```diff
@@ -6,32 +6,28 @@
...
 **Types**
 
-* *T*: any floating-point type.
-
-*Acosh* does the following with the input tensor *a*:
-
-\f[
-a_{i} = acosh(a_{i})
-\f]
+* *T*: any numeric type.
```
Comment (rkazants) [inline docs/ops/arithmetic/Acosh_3.md:27]: do we support integer type? I see there is no framework that support integer type.

Comment (pelszkow) [reply docs/ops/arithmetic/Acosh_3.md:27]: In ngraph we do: https://github.com/openvinotoolkit/openvino/blob/master/ngraph/core/src/op/acosh.cpp#L42-L53.
Also CPU tests have same as ref impl output for acosh.

**Review by ilyachur — APPROVED** [review]

**Comment by ilyachur** [comment]
@rkazants @lazarevevgeny Sorry, my mistake, I have merged this PR. If you have any comments please comment, I think we can solve these comments in the separate PR.

**Pull request merged**